src/CoreShop/Component/Address/Context/RequestBased/GeoLiteBasedRequestResolver.php

```diff
@@ -25,43 +26,87 @@ final class GeoLiteBasedRequestResolver implements RequestResolverInterface
      */
     private $countryRepository;
 
+    /**
+     * @var CoreHandlerInterface
+     */
+    private $cache;
+
+    /**
+     * @var string
+     */
+    private $geoDbFile;
+
     /**
      * @param CountryRepositoryInterface $countryRepository
+     * @param CoreHandlerInterface       $cache
      */
-    public function __construct(CountryRepositoryInterface $countryRepository)
+    public function __construct(CountryRepositoryInterface $countryRepository, CoreHandlerInterface $cache)
     {
         $this->countryRepository = $countryRepository;
+        $this->cache = $cache;
+        $this->geoDbFile = PIMCORE_CONFIGURATION_DIRECTORY . '/GeoLite2-City.mmdb';
```

Comment:
As this property is private, and there's (currently) no way to change it, why not make it a `const` instead? 

```php
private const GEO_DB_FILE = PIMCORE_CONFIGURATION_DIRECTORY . '/GeoLite2-City.mmdb';
```

Reply:
no point in that, Pimcore stores the file there as well